### DISCUSSION TITLE
Tag for {xml file name} is invalid error

### DISCUSSION THREAD

**Discussion by yashbansal130** [oppia/oppia-android#4117] [Q&A (Other)] [answered]
![Screenshot from 2022-01-20 14-15-05](https://user-images.githubusercontent.com/66557534/150336380-13031a34-c8b3-41a6-94d5-4f5208e8b965.png)
Help me with this error, how to resolve this

**Comment by FareesHussain** [comment, marked as answer]
https://github.com/oppia/oppia-android/wiki/Frequent-Errors-and-Solutions#crashing-layout-tags-in-tablet

To solve this,
1. Uninstall the app from the tablet.
2. Rebuild the app.
3. Now run the app again.
